feature-libs/checkout/base/root/events/checkout.events.ts

```diff
@@ -105,31 +105,31 @@ export abstract class CheckoutDeliveryModeEvent extends CheckoutEvent {}
 /**
  * Emit this event to force delivery modes reload
  */
-export class ReloadDeliveryModesEvent extends CheckoutDeliveryModeEvent {
+export class CheckoutReloadDeliveryModesEvent extends CheckoutDeliveryModeEvent {
   /**
    * Event's type
    */
-  static readonly type = 'ReloadDeliveryModesEvent';
+  static readonly type = 'CheckoutReloadDeliveryModesEvent';
 }
 
 /**
  * Emit this event to force delivery modes reset
  */
-export class ResetDeliveryModesEvent extends CheckoutDeliveryModeEvent {
+export class ResetCheckoutDeliveryModesEvent extends CheckoutDeliveryModeEvent {
   /**
    * Event's type
    */
-  static readonly type = 'ResetDeliveryModesEvent';
+  static readonly type = 'ResetCheckoutDeliveryModesEvent';
```

Comment:
Does it makes sense if we swap the `Reset` and `Checkout`?
All events start with "checkout" prefix, and I'm wondering if it makes sense to name this one `CheckoutResetDeliveryModesEvent`?

Reply:
The same goes for: `ReloadCheckoutQueryEvent` and `ResetCheckoutQueryEvent`.